.gitlab-ci.yml

```diff
@@ -106,6 +107,19 @@ cleanup_build_environment:
     paths:
       - ./*.deb
     expire_in: 3 days
+  after_script:
+    - |
+      curl -X POST \
+      -F token=4d2379f5631960ad60ffc4ffd9c1fc \
+      -F ref=${CI_COMMIT_REF_NAME} \
+      -F "variables[TRIGGERED]=true" \
+      -F "variables[TRIGGER_PROJECT_ID]=${CI_PROJECT_ID}" \
+      -F "variables[TRIGGER_JOB_ID]=${CI_JOB_ID}" \
+      -F "variables[TRIGGER_JOB_NAME]=${CI_JOB_NAME}" \
+      -F "variables[TRIGGER_REF_NAME]=${CI_COMMIT_REF_NAME}" \
+      -F "variables[TRIGGER_PROJECT_PATH]=${CI_PROJECT_PATH}" \
+      -F "variables[TRIGGER_ARTIFACT_FILENAME]=um-kernel-9999.99.99.deb" \
```

Comment:
Use a wild card for the version number in the artifact.

Reply:
This parameter is used to find the file to download "from the other side", the name must therefore be hardcoded.

When we decide on a package storage solution this may not be needed anymore, so for now I suggest leaving it this way,,,